editor/src/clj/editor/code/view.clj

```diff
@@ -2362,13 +2496,15 @@
 
 (defn copy! [view-node clipboard]
   (hide-suggestions! view-node)
+  (hide-hover! view-node)
```

Comment:
The call order of `hide-suggestions!` and `hide-hover!` varies. Is that intentional? I'd prefer it to be consistent otherwise.

And if it does matter, we should probably add a comment on the outlier so I don't go in and "fix it" in a fit of OCD a few years from now. 😅

Reply:
The order does not matter. I'll make the order consistent...